packages/form/tests/integration/components/cf-field/label-test.js

```diff
@@ -34,6 +34,7 @@ module("Integration | Component | cf-field/label", function (hooks) {
       { raw, owner: this.owner },
     );
     this.set("field", document.fields[0]);
+    this.set("useMandatoryAsterisk", false);
```

Comment:
That's the default, do we really need it?

Reply:
I think it's best to not explicitly set default values in tests as it makes the code less readable. To me it's much more important that the non-default value has it's dedicated and explicit test which is the case here :smile: